src/animation/AnimationMixer.js

```diff
@@ -83,8 +83,10 @@ Object.assign( THREE.AnimationMixer.prototype, THREE.EventDispatcher.prototype,
 
 		var root = optionalRoot || this._root,
 			rootUuid = root.uuid,
-			clipName = ( typeof clip === 'string' ) ? clip : clip.name,
-			actionsForClip = this._actionsByClip[ clipName ];
+			clipObject = typeof clip === 'string' ? this.getClipByName( clip ) : clip,
```

Comment:
> Considering the current functionality, and that most of it sits on the actions anyway, a per-character mixer would be a trivial wrapper binding a default for localRoot. Am I missing something?

I think that is it.  Well, except that one loads animation clips into the local-per character mixers and then play them with the simple names like "walk".  Playing the "walk" animation would resolve to different animation clips based on which per-character mixer you play it from -- because it resolves to the local per-character walk clip, a global clip named walk.  Thus it maintains the localRoot and maintains the resolving of the local animation names to their animation clips, which is usually done at load time by initializing the per-character mixer with the animation clips and their local names.  I think if we did that, such a wrapper would be equivalent to the basic per-character mixers idea.